activerecord/lib/active_record/railtie.rb

```diff
@@ -150,11 +150,12 @@ class Railtie < Rails::Railtie # :nodoc:
               next if current_version.nil?
 
               if cache.version != current_version
-                warn "Ignoring #{filename} because it has expired. The current schema version is #{current_version}, but the one in the cache is #{cache.version}."
+                warn "Ignoring #{filename} because it has expired. The current schema version is #{current_version}, but the one in the schema cache file is #{cache.version}."
                 next
               end
             end
-
+            
+            warn "Using schema cache file #{filename}"
```

Comment:
I don't think using `warn` for that make sense. What about `Rails.logger.info`?

Reply:
@byroot I switch to `puts` because `Rails.logger.info` is not yet available at the point in the startup.